Gateway/Config/PaymentPlans/PaymentPlanConfig.php

```diff
@@ -329,10 +331,23 @@ public function fixedCustomerFees(): int
     public function logoFileName()
     {
         // TODO: there's gotta be a better way
-        if (!$this->isDeferred() && in_array($this->installmentsCount(), [2, 3, 4, 10])) {
+        if (!$this->isDeferred() && in_array($this->installmentsCount(), [2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12])) {
```

Comment:
I think it is not the plugin job to filter `installmentsCount`


why did you perform this test ?